### PR TITLE
Add spaces and weight helpers documentation

### PR DESCRIPTION
**Pull request by williamcosta**
**CHANGELOG** :memo:

* Add `get-space()` helper documentation.
* Add negative number with `get-space()` helper.
* Add `get-weight()` documentation

### CODE REVIEW

File: src/scss/settings/_spaces.scss [hunk 1]
```diff
@@ -20,6 +20,32 @@
...
+*     get-spaces(box, medium) (36px)
+*     get-spaces(box, large) (48px)
+*
+*   #### Element
+*     get-spaces(element, small) (8px)
+*     get-spaces(element, medium) (12px)
+*     get-spaces(element, large) (16px)
+*
+*  @body
+*  # Nagative
```
Comment (iondrimba) [inline src/scss/settings/_spaces.scss]: @williamcosta `Negative`

File: src/scss/settings/_spaces.scss [hunk 2]
```diff
@@ -20,6 +20,32 @@
 *     .space-element-medium - 12px
 *     .space-element-large - 16px
 *
+*  @body
+*  # Helpers
+*  ##### Use the helpers to get the sizes defined in the properties such as `margin-top`.
+*
+*   @body
+*   #### Section
+*     get-spaces(section, small) (60px)
```
Comment (iondrimba) [inline src/scss/settings/_spaces.scss]: @williamcosta  `get-space`

**Comment by iondrimba** [comment]
@williamcosta descricao do PR tbm `get-space` 👍

File: src/scss/settings/_spaces.scss (hunk 1 above)
Comment (williamcosta) [reply src/scss/settings/_spaces.scss]: Ops 😁

File: src/scss/settings/_spaces.scss (hunk 2 above)
Comment (williamcosta) [reply src/scss/settings/_spaces.scss]: Ops[2] 😁

**Comment by williamcosta** [comment]
@iondrimba revised

**Pull request merged**